examples/configuration/Configuration.cs

```diff
@@ -0,0 +1,86 @@
+﻿using System;
+using NationalInstruments.SystemLink.Clients.Core;
+using NationalInstruments.SystemLink.Clients.Message;
+using NationalInstruments.SystemLink.Clients.Tag;
+
+namespace NationalInstruments.SystemLink.Clients.Examples.Configuration
+{
+    /// <summary>
+    /// Basic example console application demonstrating the different ways to
+    /// supply a configuration for SystemLink APIs.
+    /// </summary>
+    class Configuration
```

Comment:
What are your thoughts on naming the top-level example class in each application the same thing, so the user better knows which file to start at? Something like `Example.cs`

Reply:
My original thinking was if you have multiple open at once or are navigating between multiple examples, it'll be more difficult to tell where you are if all the file names are the same. I figured configuration is the only one that'll have multiple files, and maybe I should move ExampleConfiguration.cs somewhere else. I didn't give it a folder under examples because I didn't want the confusion of it being an actual example.

Maybe it could go in the root examples folder?